### PR TITLE
Feature/cookiecutter test

### PR DESCRIPTION
**Pull request by vonabarak**
# Description

Creating a new project from cookiecutter template and run pytest in it using github actions CI.

# Checklist

- [x] I've generated a new bot from the async-box template and checked that everything works:
  - [x] Linters and tests have passed
  - [x] Bot answers on `/help` command
  - [x] Bot suggests available commands
- [x] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Tests
+on: push
+jobs:
+
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
```
Comment (lxmnk) [inline .github/workflows/tests.yml]: Let's fix it to `ubuntu-20.04`.

File: scripts/test [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PRJDIR=$(pwd)
+TMPDIR=$(mktemp -d)
+cookiecutter --no-input . -o ${TMPDIR}
+
+cd "${TMPDIR}/${BOT_NAME}" || exit 1
+python -m venv .venv
+source .venv/bin/activate
+pip install poetry
+poetry install
+APP_ENV=test pytest --cov-config=setup.cfg
```
Comment (lxmnk) [inline scripts/test:21]: Pls use `APP_ENV=$(test pytest --cov-config=setup.cfg)` form.

Comment (vonabarak) [reply scripts/test:21]: It is not a variable assigning. 
It is just running pytest with APP_ENV set to "test".

File: .github/workflows/tests.yml (hunk 1 above)
Comment (vonabarak) [reply .github/workflows/tests.yml]: fxd

**Review by lxmnk — APPROVED** [review]

**Review by Dnomin — APPROVED** [review]

**Review by YunusovSamat — APPROVED** [review]

**Review by Opti213 — APPROVED** [review]

**Pull request merged**